src/app/public/modules/flyout/flyout.component.spec.ts

```diff
@@ -20,7 +20,8 @@ import {
 } from '@skyux-sdk/testing';
 
 import {
-  Observable
+  Observable,
+  throwError
```

Comment:
We can't migrate to RxJS 6 just yet, because it wouldn't work with SKY UX 2 SPAs.

Reply:
I've reverted to the correct import for compatibility and have added a note to do the change later